plugins/channelrx/localsink/localsinksettings.cpp

```diff
@@ -53,6 +53,7 @@ void LocalSinkSettings::resetToDefaults()
     m_reverseAPIChannelIndex = 0;
     m_workspaceIndex = 0;
     m_hidden = false;
+    m_maxFFTBands = 32;
```

Comment:
Why 32 and not leave it at 20?

Reply:
Sorry, leftovers from experiments :) I'll undo the change.